Patch-Source: https://github.com/tomerfiliba/reedsolomon/pull/38

From 63e5bd9fc3ca503990c212eb2c77c10589e6d6c3 Mon Sep 17 00:00:00 2001
From: Marco Rossi <developer@marco-rossi.com>
Date: Tue, 23 Nov 2021 20:31:50 +0100
Subject: [PATCH] Force cythonize of *.pyx file

If not set the also exisiting and *.c file is used
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 63d4264..ca08d02 100644
--- a/setup.py
+++ b/setup.py
@@ -31,7 +31,7 @@
     # If Cython is installed, transpile the optimized Cython module to C and compile as a .pyd to be distributed
     from Cython.Build import cythonize
     print("Cython is installed, building creedsolo module")
-    extensions = cythonize([ Extension('creedsolo', ['creedsolo.pyx']) ])
+    extensions = cythonize([ Extension('creedsolo', ['creedsolo.pyx']) ], force=True)
 except ImportError:
     # Else Cython is not installed (or user explicitly wanted to skip)
     if '--native-compile' in sys.argv:
